### PR TITLE
Rename flag OJDKTHREAD_SUPPORT to OPENJDK_THREAD_SUPPORT

### PR DESCRIPTION
**Pull request by keithc-ca**
As discussed: https://github.com/ibmruntimes/openj9-openjdk-jdk/pull/387#discussion_r779782811.

Should be merged concurrently with https://github.com/ibmruntimes/openj9-openjdk-jdk/pull/402.

### CODE REVIEW

**Comment by keithc-ca** [comment]
FYI @JasonFengJ9

File: buildspecs/j9.flags [hunk 1]
```diff
@@ -1832,7 +1832,7 @@ Only available on zOS</description>
 		<description>Enables support for Foreign Linker API (part of Project Panama) such as native invocations to ffi_call</description>
 		<ifRemoved>No support for Foreign Linker API</ifRemoved>
 	</flag>
-	<flag id="opt_ojdkThreadSupport">
+	<flag id="opt_openjdkThreadSupport">
 		<description>Enables OpenJDK Thread support for Project Loom</description>
 		<ifRemoved>No OpenDJK Thread support for Project Loom</ifRemoved>
```
Comment (JasonFengJ9) [inline buildspecs/j9.flags:1837]: Since we are here, please help fix this typo, it should be `OpenJDK`.

Comment (keithc-ca) [reply buildspecs/j9.flags]: I should have noticed and fixed that without your prompt: fixed.

**Review by JasonFengJ9 — APPROVED** [review]
LGTM

**Comment by pshipton** [comment]
jenkins compile alinux64 jdk8

**Review by pshipton — APPROVED** [review]

**Pull request merged**